### PR TITLE
Add KEGGCharter

### PR DESCRIPTION
**Pull request by iquasere**
Describe your pull request here

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by iquasere** [comment]
@bioconda-bot please fetch artifacts

**Comment by iquasere** [comment]
@bioconda-bot please add label

**Comment by iquasere** [comment]
@bioconda-bot please merge

**Comment by dpryan79** [comment]
@iquasere Now you'll be able to merge it, you just need someone else to approve :)

**Comment by iquasere** [comment]
> @iquasere Now you'll be able to merge it, you just need someone else to approve :)

I'm still a noob at this, sorry for the spam xD

**Comment by mbargull** [comment]
> @dpryan79 I'm still a noob at this, sorry for the spam xD

Better to try things out than not ;).

File: recipes/keggcharter/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  version: {{ version }}
+
+source:
+  url: https://github.com/iquasere/KEGGCharter/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: generic
+  number: 0
+  script: "mkdir -p $PREFIX/bin && cp *.py KEGGCharter_prokaryotic_maps.txt $PREFIX/bin"
```
Comment (mbargull) [inline recipes/keggcharter/meta.yaml:16]: Ideally, we'd have only executables in `${PREFIX}/bin`.
As a start, you could do something like
```suggestion
  script: |
    dir="${PREFIX}/share/KEGGCharter"
    mkdir -p "${dir}"
    cp *.py KEGGCharter_prokaryotic_maps.txt "${dir}/"
    mkdir -p "${PREFIX}/bin"
    chmod +x "${dir}/kegg_charter.py"
    ln -s "${dir}/kegg_charter.py" "${PREFIX}/bin/"
```
(Or you could turn this into a Python package and let `setuptools`/`pip` (or another build tool of your choice!) handle these kinds of things, of course ;).)

Comment (iquasere) [reply recipes/keggcharter/meta.yaml:16]: I have never used package managers, tho I feel is becoming necessary when submitting tools to bioconda... for now this approach of putting everything in bin was doing the trick, I didn't know you wanted only binaries in bin, but it makes sense xD I like this alteration, where could setuptools or pip help in it?

**Comment by iquasere** [comment]
@bioconda-bot please merge

**Comment by iquasere** [comment]
@mbargull changes were not applied to the [commit](https://github.com/bioconda/bioconda-recipes/blob/master/recipes/keggcharter/meta.yaml)? In the next version, which will come out in the next days, I'll rectify this

**Pull request merged**